tests/test_DataFrameWrapper.py

```diff
@@ -0,0 +1,211 @@
+from chispa.dataframe_comparer import assert_df_equality
+from pyspark.sql import SparkSession
+from pyspark.sql import DataFrame as SparkDataFrame
+from pyspark.sql.functions import col
+from pyspark.sql.types import StructType, StructField, StringType
+from pytest import fixture
+from pandas import DataFrame
+
+from sds_data_model.dataframe import DataFrameWrapper
+
+# Create dataframe to test against
+expected_data = {'a': [1, 2, 3],
+                 'b': [3, 4, 5],
+                 'c': [5, 6, 7]}
+expected_df = DataFrame(expected_data)
+
+
+@fixture
+def spark_session() -> SparkSession:
+    """Local Spark context."""
+    return (
+        SparkSession.builder.master(
+            "local",
+        )
+        .appName(
+            "Test context",
+        )
+        .getOrCreate()
+    )
+
+
```

Comment:
- As mentioned above, I'd cut out the middleman and create a Spark DataFrame directly.
- This means moving your `expected_schema` definition forward
- Also, as your expected DataFrame has integer we should really be testing for integers. See below for how to stop Spark from treating the columns as strings. 

```suggestion


@fixture
def expected_schema() -> StructType:
    return StructType(
        [
            StructField("a", IntegerType(), True),
            StructField("b", IntegerType(), True),
            StructField("c", IntegerType(), True),
        ]
    )


@fixture
def expected_dataframe(
    spark_session: SparkSession,
    expected_schema: StructType,
) -> SparkDataFrame:
    data = [
        {"a": 1, "b": 4, "c": 7},
        {"a": 2, "b": 5, "c": 8},
        {"a": 3, "b": 6, "c": 9},
    ]
    return spark_session.createDataFrame(
        data=data,
        schema=expected_schema,
    )


```